bluesky/tests/test_persistent_dict.py

```diff
@@ -20,13 +21,35 @@ def test_persistent_dict(tmp_path):
     # Update a value and check again.
     d['a'] = 2
     expected = dict(d)
+
+    # Force sync changes from d to actual
+    d.flush()
```

Comment:
This is technically not needed because we are setting a top level key, right?

Reply:
You're right, just the reload is needed.